clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -52,42 +46,34 @@ jest.mock("vscode", () => ({
     }),
   },
   workspace: {},
+  window: {
+    showErrorMessage: jest.fn().mockReturnValue(""),
```

Comment:
just curious, what code triggered the `showErrorMessage` and why?

Reply:
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2265/files/8ce0f8515a9a2385de1d9abbff255fdab177093a

this unused method was triggering. It is deleted now. I forgot to remove the mock. here should be resolved : 9be6c0a07652acd9800635c19569de87906bf6ce